.storybook/preview.js

```diff
@@ -16,20 +16,41 @@ const preview = {
         date: /Date$/,
       },
     },
+    viewport: {
+      viewports: {
```

Comment:
I use to have several mobile devices to choose from, with this I only have two.  Do you really need this? If so, you can move it to the export default in your story.   

Also, most of my responsiveness testing is done by dragging the side of the storybook window to make it bigger and smaller and watch it handle the transition.  

See stories/footer-stories.jsx for an example of putting viewport in the story, and even changing the default viewport per story.